rustls/src/crypto/mod.rs

```diff
@@ -588,6 +566,46 @@ pub fn default_fips_provider() -> CryptoProvider {
     aws_lc_rs::default_provider()
 }
 
+mod crypto_default_provider {
+    use alloc::sync::Arc;
+
+    #[cfg(not(feature = "std"))]
```

Comment:
Keep both `alloc` impls together (no empty line), please, even with the guard.

Reply:
done